Source/Core/Core/CoreTiming.cpp

```diff
@@ -49,14 +51,14 @@ static void EmptyTimedCallback(Core::System& system, u64 userdata, s64 cyclesLat
 //
 // Technically it might be more accurate to call this changing the IPC instead of the CPU speed,
 // but the effect is largely the same.
-int CoreTimingManager::DowncountToCycles(int downcount) const
+s64 CoreTimingManager::DowncountToCycles(s64 downcount) const
 {
-  return static_cast<int>(downcount * m_globals.last_OC_factor_inverted);
+  return downcount * m_globals.last_OC_factor_inverted;
 }
 
-int CoreTimingManager::CyclesToDowncount(int cycles) const
+s64 CoreTimingManager::CyclesToDowncount(s64 cycles) const
 {
-  return static_cast<int>(cycles * m_last_oc_factor);
+  return cycles * m_last_oc_factor;
 }
```

Comment:
I can't see any reason you've changed these from `int` to `s64`. This is particularly confusing because the actual `downcount` variable (in `ppcState`) remains an `int` -- and since it's accessed directly by the JIT it probably should stay as such; if you want to change this make a separate PR and state your reason.

I can perhaps see a reason to change `DowncountToCycles()` to return an `s64` if you're afraid it will over/underflow, but I'm not sure how likely that is in practice...?

Reply:
your probably right, it just was kinda buggin me how some of the vars were int and some of them were s64, I just made them all s64 to be consistent but your right in that it doesnt make sense to do that in this PR.